browser/plugins/three_hemisphere_light.plugin.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+	var ThreeHemisphereLightPlugin = E2.plugins.three_hemisphere_light = function(core) {
+		ThreeObject3DPlugin.apply(this, arguments)
+
+		this.desc = 'THREE.js Hemisphere Light'
+
+		this.params = {
+			skyColor: new THREE.Color(0xffffff),
+			groundColor: new THREE.Color(0x684f40),
+			intensity: 1.0
+		}
+
+		this.input_slots = [
+			{ name: 'intensity', dt: core.datatypes.FLOAT, def: this.params.intensity },
+			{ name: 'color', dt: core.datatypes.COLOR, def: this.params.skyColor },
```

Comment:
how can we eg. modulate `color` here? there is no `update_input()` or `update_state()`?